experiment/simulation/index.html

```diff
@@ -1,43 +1,185 @@
+                    <label class="v-checkbox" style="align-items:flex-start; flex: 50%; display: flex;">
+                        <input type="checkbox" id="xz-grid-cb">
+                        <span style="font-size:x-small;">XZ-Grid </span>
+                    </label>
+                </div>
+
+
+                <div class="v-datalist-title" id="top-border">Camera Options</div>
+                <p>
+
+                    <button id="add-shape-btn" style="margin:0.25rem;" class="v-button is-small is-responsive is-info" onclick="addShape()">Add</button>
+                    <button id="edit-shape-btn" style="margin:0.25rem;" class="v-button is-small is-responsive  is-success" onclick="editShape()">Edit</button>
+                    <button id="delete-shape-btn" style="margin:0.25rem;" class="v-button is-small is-responsive  is-danger" onclick="deleteShape()">Delete</button>
+
+                    <div id="edit-modal" class="modal is-responsive">
+                        <div class="modal-content">
+                            <span class="close">&times;</span>
+                            <p>
+                                <p class="v-datalist-title"> Edit Shape Menu</p>
+                                <div id="Addshape" class="control" value="hi">
+                                    <div class="select" style="float:left;">
+                                        <select name="" id="shape-edit-dropdown" value="dropdown">
+                                    <option value="Tetrahedron">Tetrahedron</option>
+                                    <option value="Octahedron">Octahedron</option>
+                                    <option value="Dodecahedron">Dodecahedron</option>
+                                    <option value="Cube">Cube</option>
+                                </select>
+                                    </div>
+                                </div>
+                                </br>
+                                </br>
+
+                                <input class="input " style="width: 200px; float:left" id="x" type="number" placeholder="X" step=".01" value="0">
+                                <input class="input " style="width: 200px; float:left" id="y" type="number" placeholder="Y" step=".01" value="0">
+                                <input class="input " style="width: 200px; float:left" id="z" type="number" placeholder="Z" step=".01" value="0">
+
+                                <div class="control">
+                                    <button class="buttonisprimary" style="width: 200px; float:left; margin: 1rem;">Edit</button>
+                                </div>
+                            </p>
+                        </div>
+
+                    </div>
+
+                    <div id="add-modal" class="modal is-responsive">
+                        <div class="modal-content">
+                            <span class="close">&times;</span>
+                            <p>
```

Comment:
Why are we using so many paragraphs? Please use [semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp)  to delineate the structure of your document in place of using non-semantic elements.
[Here](https://medium.com/before-semicolon/10-html-semantic-tags-and-when-to-use-them-5ae7d7d0b0f2) is a good reference

Reply:
used the main, aside and section tags as well. modified/removed the p tags, wherever unnecessary